template/blt/project.yml

```diff
@@ -39,15 +44,15 @@ target-hooks:
   # Executed when front end dependencies should be installed.
   frontend-setup:
     # E.g., ${docroot}/sites/all/themes/custom/mytheme.
-    dir: ${docroot}
+    dir: '${docroot}/themes/custom/${project.subtheme.machine_name}'
     # E.g., `npm install` or `bower install`.
-    command: echo 'No frontend-setup configured.'
+    command: 'npm install -g yarn; yarn install;'
   # Executed when front end assets should be generated.
   frontend-build:
     # E.g., ${docroot}/sites/all/themes/custom/mytheme.
-    dir: ${docroot}
+    dir: '${docroot}/themes/custom/${project.subtheme.machine_name}'
```

Comment:
The `frontend:cog:init` theme could just write this value using `drupal yaml:update` so that it doesn't exist prior to the directory existing.